MLH-Fellowship/cli
# Implemented command to download the Hermes Sampling Profiler to a local machine

## Summary
- Solves #1 
- Add ```packages/cli/src/commands/profile```: a folder for Hermes Sampling Profile related commands
- File ```packages/cli/src/commands/profile/index.ts```: exporting the default command ```profile-hermes [destinationDir] --fileName [string]```
- File ```packages/cli/src/commands/profile/downloadProfile.ts```: implement the functions that help with the above command 👆 

- Command added in the file ```packages/cli/src/commands/index.ts```: add the profile command in the list of ```projectCommands```

## Test Plan:
- I first followed this guide on how to test changes [Testing changes](https://github.com/MLH-Fellowship/cli/blob/master/CONTRIBUTING.md#testing-your-changes).
- After that, the usage of the command is:
   - Navigate to the directory of a React Native project 
   -  Run the command ```profile-hermes [destinationDir] --fileName [string]``` with appropriate inputs
       - ```destinationDir```: is optional, provided it with the directory where users want to pull the profile to (if not present, pull to the current React Native root directory)
      - ```--fileName [string]``` an option to specify which profile users want to pull (if not present, pull the latest profile)

- Make sure your emulator is running and adb is connected
